continuous-integration/linux/lint.sh

```diff
@@ -50,7 +53,7 @@ runShellcheck() {
     echo ""
     echo "Running shellcheck on:"
     echo "${fileList}"
-    shellcheck -x -e SC1090,SC2086,SC2046 $fileList || exit $?
+    shellcheck -x -e SC1090,SC2086,SC2046 --source-path=${SCRIPT_DIR} $fileList || exit $?
```

Comment:
What is this change about, if I may ask? It is not mentioned in the commit message.

Reply:
It's so that I'm allowed to do `source $SCRIPT_DIR/venv.sh` without shellcheck freaking out. 
I'm basically telling shellcheck that it's okay to source files from the directory.